skins/cat17/src/app/lib/view_handler/section_info.js

```diff
@@ -227,7 +227,11 @@ var objectAssign = require( 'object-assign' ),
 				this.setSectionStatus( SECTION_STATUS.disabled );
 			}
 
-			this.setIcon( this.getValueIcon( addressType ) );
+			if ( aggregateValidity.dataEntered ) {
+				this.setIcon( this.getValueIcon( addressType ) );
+			} else {
+				this.setIcon( undefined );
+			}
```

Comment:
This seemed like something that should go into its own method (to not mix levels of abstraction), so I checked the whole `DonorTypeSectionInfo` class(?). Hello one big method with over9000 parameters :( Not obvious to me how to deal with the long parameter list, though a lot of things could be extracted from the big method. Perhaps something to do after this PR is merged. Idk how to best do such extraction in this type of JS, so am curious to see.